data/card_catalogue/upgradeable-1.json

```diff
@@ -234,7 +303,10 @@
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "All players will critically succeed on their next d20 roll this turn.",
+      "rules": {
+        "active": "Cause all players to critically succeed on their next d20 roll.",
```

Comment:
I'm not opposed to some ridiculous attempt to sustain the effect of this card by not doing anything at all.

Reply:
this comment made irrelevant pending 
- #124